server/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 alpine:latest
+
+# Update package list and install required certificates
+RUN apk --no-cache add ca-certificates \
+    && apk add --no-cache libc6-compat
+
+# Set a working directory (optional)
+WORKDIR /app
+
+# Copy the local binary into the container
+COPY ./deweb-server_linux_amd64 /app/deweb-server_linux_amd64
+
+# Give execution permissions to the binary
+RUN chmod +x /app/deweb-server_linux_amd64
+
+# Expose the port on which the server listens (optional, if needed)
+EXPOSE 8080
```

Comment:
Users might want to use a specific DeWeb server configuration and so, use a different port, is that something that can be handled ? Or should users using docker change the exposed port another way than the DeWeb server config file ?

Reply:
That's why docker is so cool. 

EXPOSE here say, IN the docker container we want to expose the port 8080 (because i know my software use this port), OTHER ports can't be acceded. 

user can choose to use any port on their host machine ( ex with -p option) 

ex user want to run deweb_server on port 5000 : 

`docker run -p 5000:8080 deweb_server`